src/Output/Output_DumpData_Total_HDF5.cpp

```diff
@@ -280,10 +285,17 @@ void Output_DumpData_Total_HDF5( const char *FileName )
    int  NFieldStored = 0;
 
    const int FluDumpIdx0 = NFieldStored;
-   NFieldStored += NCOMP_TOTAL;
-   if ( FluDumpIdx0+NCOMP_TOTAL-1 >= NFIELD_STORED_MAX )
+#  if (  ELBDM_SCHEME == ELBDM_HYBRID  &&  !defined( GAMER_DEBUG )  )
+   const int NCompStore  = NCOMP_TOTAL - 1;  // do not store STUB field unless we are in debug mode
+#  else
+   const int NCompStore  = NCOMP_TOTAL;
+#  endif
+
+   NFieldStored += NCompStore;
+
+   if ( FluDumpIdx0+NCompStore-1 >= NFIELD_STORED_MAX )
       Aux_Error( ERROR_INFO, "exceed NFIELD_STORED_MAX (%d) !!\n", NFIELD_STORED_MAX );
-   for (int v=0; v<NCOMP_TOTAL; v++)   sprintf( FieldLabelOut[ FluDumpIdx0 + v ], "%s", FieldLabel[v] );
+   for (int v=0; v<NCompStore; v++)    sprintf( FieldLabelOut[ FluDumpIdx0 + v ], "%s", FieldLabel[v] );
```

Comment:
[Style]
```suggestion
   for (int v=0; v<NCompStore; v++)   sprintf( FieldLabelOut[ FluDumpIdx0 + v ], "%s", FieldLabel[v] );
```

Reply:
This is fine as I usually align the beginning of a statement to 3n+1 spaces since GAMER adopts a three-space indentation style.